lib/command.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+var mson2json = require('./mson2json');
+var fs        = require('fs');
+var path      = require('path')
+
+var M2jCommand = (function() {
+    function M2jCommand(options, cb) {
+        if (options == null) {
```

Comment:
if checking for equality, use the exact match operator `===`, otherwise this will eval to true for all falsy values (ie empty string, false, undefined, null, 0, etc)